packages/debugger/test/helpers.js

```diff
@@ -25,7 +25,7 @@ export async function prepareContracts(provider, sources = {}, migrations) {
 
   config.compilers = {
     solc: {
-      version: "0.8.9",
+      version: "0.8.12",
       settings: {
         optimizer: { enabled: false, runs: 200 },
```

Comment:
This is not part of this PR's changes but I'm curious what this optimizer option does, enabled is false **and** runs is 200?